.github/workflows/latest-release.yml

```diff
@@ -57,3 +57,38 @@ jobs:
       - name: Push discovery engine images to Docker
         run: |
             ./src/build/push_knoxautopolicy.sh ${{ steps.vars.outputs.tag }}
+
+  push-stable-version:
+    name: Create discovery engine Release - 18.04
+    if: github.repository == 'accuknox/discovery-engine'
```

Comment:
We won't need this as we are already checking this in `build`, instead add a check for `if: github.ref != 'refs/heads/main'` so that this action, works only on that branch